models/intel/smartlab-action-recognition-0001/composite-model.yml

```diff
@@ -0,0 +1,5 @@
+description: >-
+  General-purpose action recognition model for DSI1867 dataset
```

Comment:
please remove all mentions of internal dataset ID!!!

Reply:
fixed